install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -11,122 +11,364 @@
+statisfies the pod, and, thus, the pod will start.
+
+==== Custom SCC for Group IDs
+If GID range checking is desired, and none of the predefined SCCs are to be edited, then the
+custom link:#nfs-scc["nfs-scc"] can be modified to support this reqirement, as seen below:
+```
+#after: oc edit scc nfs-scc
+...
+# oc export scc nfs-scc 
+...
+kind: SecurityContextConstraints
+metadata:
+...
+  name: nfs-scc
+priority: 9
+...
+supplementalGroups:
+  ranges: <2>
+  - max: 6000
+    min: 5000
+  type: MustRunAs <1>
+```
+<1> _MustRunAs_ triggers gid range checking
+<2> the min and max values are defined in the SCC, therefore the SCC statisfies the need for
+range checking, and thus the namespace's `openshift.io/sa.scc.supplemental-groups` range is
+not needed.
+
+Only the "nfs-scc" has been changed. The pod spec (fragment shown just above) does not need any
+changes, and the _default_ namespace also remains the same. After creating the pod:
+```
+# oc create -f nfs-pod.yaml
+...
+# oc get pod nfs-pod1 -0 yaml
+...
+metadata:
+  annotations:
+    openshift.io/scc: nfs-scc <1>
+  name: nfs-pod1
+  namespace: default <2>
+...
+spec:
+  containers:
+    ...
+    securityContext:
+      runAsUser: 1000000000 <3>
+ ...
+ securityContext:
+    seLinuxOptions:
+     level: s0:c1,c0
+    supplementalGroups:
+    - 5555 <3>
+...
+# oc rsh nfs-pod1 id
+uid=1000000000 gid=0(root) groups=5555 <3>
+```
+<1> verifying that the "nfs-scc" matched the pod.
+<2> the namespace is _default_
+<3> verifying that the running container has the default user id (1000000000) and
+supplemental groups of 5555
+
+=== SELinux
+*NOTE:* The
+link:pod_security_context.html#selinux[pod and storage security] document covers
+controlling storage access in conjunction with using selinux.
+
+All predefined SCCs, except for the _privileged_ SCC, set the `seLinuxContext` to _MustRunAs_.
+This forces the pod to use a selinux policy, which can be defined in the pod spec. See the
+fragment below:
+```
+spec:
+  containers:
+  - name: ...
+  securityContext:
+    type: MustRunAs
+    SELinuxOptions: <1>
+      user: selinux-user-name
+      role: selinux-role-name
+      type: selinux-type-label
+      level: selinux-level
+```
+<1> selinux policy here is defined in the context of a container, but it can also be set global
+to the pod.
+
+If selinux is not supplied in the pod spec then it defaults to the selinux policy defined in the
+pod's matching SCC, or, if absent in the SCC, then the namespace's `sa.scc.mcs:` value is used. 
+
+*Note:* if pods are not authorizing, or if the NFS mount is failing due to permissions errors, then
+there is a possibility that selinux enforcement is interfering. One way to check for this is to run:
+```
+ausearch -m avc --start recent
+```
+which examines the log file for AVC (Access Vector Cache) errors.
+
+It is also usually necessary to enable *virt_sandbox_use_nfs*, as seen below:
```

Comment:
@screeley44 I'll look at the selinux portion of the persistent_storage_nfs doc to make sure I'm not repeating too much.